problemsets-mat223/linearalbegra.tex

```diff
@@ -973,6 +973,13 @@ \section*{Lines and Planes}
 	Let $A$ be the set of points $(x,y)\in\R^2$ such that $y=2x+1$.
 	\begin{parts}
 		\item Describe $A$ using set-builder notation.
+			\begin{solution}
+				$\Set*{\vec v \in \R^2 \given \vec v = \begin{bmatrix*}[c] x \\ 2x+1\end{bmatrix*} \text{ for some } x \in \R}$
```

Comment:
Use `\mat{...}` instead of `\begin...\end` There should be a `\matc` if you want it centered.

Reply:
Cool. Didn't know about \matc. Will do.